src/common/common.py

```
@@ -405,6 +405,23 @@ def page_setup(page: str = "") -> dict[str, Any]:
                 width=1,
                 height=1,
             )
+        if (st.session_state.settings["analytics"]["matomo"]["enabled"]) and (
+            st.session_state.tracking_consent["matomo"] == True
```

Ruff (0.15.1)

[error] Avoid equality comparisons to `True`; use `st.session_state.tracking_consent["matomo"]:` for truth checks

Replace with `st.session_state.tracking_consent["matomo"]`

(E712)